tinygrad/codegen/linearizer.py

```diff
@@ -218,13 +218,14 @@ def calc_tc_idxs(local_sizes: List[int], aliases: List[List[int]]):
     return alias_buf_idxs
 
   def render_reduceop(self, reduceop:LazyOp, accs:Dict[LazyOp, List[UOp]], loaded_buffers:Dict[Union[MemBuffer, ConstBuffer, LocalBuffer], List[UOp]],
-                      global_idxs, local_idxs, upcast_idxs, full_upcast_idxs, reduce_idxs, fake_reduce_idxs,
+                      global_idxs, local_idxs, upcast_idxs, full_upcast_idxs, reduce_idxs,
                       alias_buf_idxs:List[Tuple[int, int, List]]) -> Tuple[List[NumNode|Variable], List[NumNode|Variable]]:
+    fake_reduce_idxs = [x*0 for x in reduce_idxs]
```

Comment:
@0xtimmy what's the reason for this?

Reply:
I originally didn't pass them through `render_reduceop` but I can